internal/operands/common-templates/reconcile_test.go

```diff
@@ -32,23 +33,18 @@ const (
 	namespace = "kubevirt"
 	name      = "test-ssp"
 
-	testOsLabel       = TemplateOsLabelPrefix + "some-os"
-	testFlavorLabel   = TemplateFlavorLabelPrefix + "test"
-	testWorkflowLabel = TemplateWorkloadLabelPrefix + "server"
-	futureVersion     = "v999.999.999"
+	futureVersion = "v999.999.999"
 )
 
-func TestTemplates(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "Common Templates Suite")
-}
-
 var _ = Describe("Common-Templates operand", func() {
+	const defaultArch = runtime.GOARCH
```

Comment:
I'd rather set this to a fixed value. Tests should behave the same on all arches.

Reply:
Possible you could iterate over all possible / supported values?

Reply:
In `ssp-operator/internal/operands/common-templates/reconcile.go` the function `getClusterArchitectures()` returns `runtime.GOARCH` if nothing else is specified in the SSP object. It is to preserve backward compatibility on clusters different than `amd64`.
Unit tests only copy this logic. 

But I agree, that unit tests should be independent of the arch. I will think how to fix this.

Reply:
done